config/routes.rb

```diff
@@ -8,6 +8,6 @@
 
     get "translate", to: "translations#index"
     get "about", to: "static#about"
-    root to: "doctors#search"
+    root to: "doctors#search_root"
```

Comment:
I think a good, progressive enhancement approach would be to add the button to the page with jquery if the search form has a class like ‘search-with-locate-button’. That way browsers without js don't get it.

Then we just add that class to the form on pages where we want the locate button.

That’s quite flexible then, and it’s not dependent on the Controller/route which might change.